src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  const thumb = eventElement.target.closest('.gallery__thumb');
+
+  if(!thumb) {
+
+    eventElement.preventDefault();
+    const largePrevious = document.querySelector('.gallery__large-img');
+
+    largePrevious.src = eventElement.target.src;
```

Comment:
This line of code will only execute if the clicked element is not a thumbnail (because of the inverted `if` condition). However, you probably want to change the source of the large image when a thumbnail is clicked. If that's the case, this line should be inside the `if` block (when `thumb` is truthy), not outside of it.